tests/yast2_gui/yast2_security.pm

```diff
@@ -38,6 +39,11 @@ sub run {
     wait_screen_change { type_string "30" };
     wait_screen_change { send_key "alt-o" };
 
+    reset_consoles;
+    wait_still_screen 10;
+
+    handle_logout;
+    handle_login;
```

Comment:
why to logout and login back? Please in general explain all your changes in the description of the PR and commit.

Reply:
Done.